frame/wormhole/issuing/s2s/src/lib.rs

```diff
@@ -97,48 +104,40 @@ pub mod pallet {
 		#[transactional]
 		pub fn register_from_remote(
 			origin: OriginFor<T>,
-			token: Token,
+			token_metadata: TokenMetadata,
 		) -> DispatchResultWithPostInfo {
 			let user = ensure_signed(origin)?;
-			ensure_source_root::<T::AccountId, T::BridgedAccountIdConverter>(
```

Comment:
I guess `ensure_source_root` is not used anymore. please delete the implementation in primitives.

Reply:
Last time we had discussed this implementation. The conclusion is to keep it here.